Sources/ManageMaintenance.php

```diff
@@ -854,14 +854,14 @@ function AdminBoardRecount()
 		{
 			// Recount approved messages
 			$request = $smcFunc['db_query']('', '
-				SELECT /*!40001 SQL_NO_CACHE */ t.id_topic, MAX(t.num_replies) AS num_replies,
-					CASE WHEN COUNT(ma.id_msg) >= 1 THEN COUNT(ma.id_msg) - 1 ELSE 0 END AS real_num_replies
+				SELECT t.id_topic, MAX(t.num_replies) AS num_replies,
+					GREATEST(COUNT(ma.id_msg) - 1, 0) AS real_num_replies
```

Comment:
Just curios if its safe to use GREATEST() with our current min mysql version.  It seems its behavior changed a bit: https://bugs.mysql.com/bug.php?id=28258

Reply:
Since aggration function can't return null and 0 can't be null,
so we don't need to care about this topic.